### DISCUSSION TITLE
Default map schematic files for oneblock would be appreciated!

### DISCUSSION THREAD

**Discussion by 8MiYile** [BG-Software-LLC/SuperiorSkyblock2#2222] [Pending Feature Requests] [answered]
because idk why i can't create oneblock skyblock world

**Comment by Lemvs** [comment, marked as answer]
You just need to place any block, such as a Grass Block, anywhere, type /is admin schematic, and right-click and left-click that block while holding a Golden Axe; then, stand on that block and type /is admin schematic <name>, and that's all.

@OmerBenGera you can mark this answered and close it.
